source/l4_testing/hal/stm32l4xx_hal_conf.h

```diff
@@ -0,0 +1,479 @@
+/********************************************************************************
```

Comment:
Do we use the STM32 HAL at all for this project anymore?

Reply:
No, forgot it was there. I will remove it.